src/drawer.multicanvas.js

```diff
@@ -52,28 +52,30 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.MultiCanvas, {
             this.removeCanvas();
         }
 
+        var leftOffset = 0;
         for (var i in this.canvases) {
             // Add some overlap to prevent vertical white stripes, keep the width even for simplicity.
-            var canvasWidth = this.maxCanvasWidth + 2 * Math.ceil(this.params.pixelRatio / 2);
+            var canvasWidth = Math.floor(totalWidth / requiredCanvases * this.params.pixelRatio),
+                canvasOverlap = 2 * Math.ceil(this.params.pixelRatio / 2),
+                actualCanvasWidth = canvasWidth;
```

Comment:
This is the main part of the change. Make the canvases equal width (or very close, the last one will be slightly smaller as it doesn't have an overlap).